src/app/pipes/tooltipdisplay.ts

```diff
@@ -0,0 +1,13 @@
+import { PipeTransform, Pipe } from '@angular/core';
+
+@Pipe({name: 'tooltipDisplay'})
+export class TooltipDisplayPipe implements PipeTransform {
```

Comment:
This looks like it could be useful to simplify the ugly `ngIf` in `table.component.html`:

```html
<ng-container *ngIf="linkColumnMappings &&
                    uidColumn &&
                    linkColumnMappings[item[uidColumn]] !== undefined &&
                    linkColumnMappings[item[uidColumn]][column.key] !== undefined;
                    else notLinkColumn">
```

Could you rename this pipe to something like `tableColumn`? Then you can rename `uid` to `row`. Then replace that ugly `ngIf` with this pipe.